### ISSUE TITLE
SimpleLogic wires make C&B multiparts invisible

### ISSUE THREAD

**Issue by COLDW0LF** [asiekierka/CharsetMC#361]
Placing SimpleLogic wires in the same block as C&B blocks is causing some issues for me.
More than likely an MCMultipart issue, but wanted to let you know!

No errors are getting thrown/logging that I'm aware of. 
Fancy render is enabled, no Optifine. 
MAGE, VanillaFix, FoamFix, and BetterFPS are being used.

- If C&B block is placed first, wires will place perfectly. Upon re-loading the world, C&B are invisible.
- If wires are placed first, C&B blocks will either refuse to place, or will place and become invisible.
- Wires remain functional regardless of the rendering state of the multipart.

Before Load: https://i.imgur.com/rMXOS6h.png
After Load: https://i.imgur.com/BIc0dhb.png

**Comment by asiekierka** [comment]
Throw out VanillaFix and BetterFPS first. As the developer, I'm fairly confident FoamFix and MAGE shouldn't cause issues here.

I assume you have the absolute latest version of every mod?

**Comment by COLDW0LF** [comment]
Yep, everything's up to date. I'll try removing them both now and let you know if that changes anything!

Update: 
Tried without all four just to be thorough; issue is still present, unfortunately.

**Comment by COLDW0LF** [comment]
I don't know if it's related but I just had a crash related to Multipart while placing wires
https://paste.dimdev.org/alanavosag.mccrash
